help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,142 @@
+
+### Use case
+
+Marketing teams can add new customer contacts into CRM through Adobe Experience Platform, build segments from their own offline data, and send these segments to Salesforce CRM. Once contacts are added they can be used in various ways. For example sending emails for marketing campaigns or to display ads in their social media feeds.
+
+## Prerequisites {#prerequisites}
+
+Before activating data to the Salesforce CRM destination, you must have a [schema](https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+The following items are required before you start configuring the destination.
+
+1. You need to have a Salesforce account.
+    * Go to the Salesforce [trial](https://www.salesforce.com/in/form/signup/freetrial-sales/) page to register and create a Salesforce account, if you do not have one already.
+1. Note down your [Salesforce domain](https://help.salesforce.com/s/articleView?id=sf.domain_name_setting_login_policy.htm&type=5).
+1. Create a [connected app](https://help.salesforce.com/s/articleView?id=sf.connected_app_create.htm&language=en_US&r=https%3A%2F%2Fhelp.salesforce.com%2F&type=5), enable its [OAuth Settings](https://help.salesforce.com/s/articleView?id=connected_app_create_api_integration.htm&type=5&language=en_US) and once created successfully note down its *Consumer Key* and *Consumer Secret*. Refer to the [Salesforce documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_rotate_consumer_details.htm&type=5) if you need additional guidance.
+1. Select the following [scopes](https://help.salesforce.com/s/articleView?id=connected_app_create_api_integration.htm&type=5&language=en_US) for the connected app in the Salesforce user interface:
+    - chatter_api
+    - lightning
+    - visualforce
+    - content
+    - openid
+    - full
+    - api
+    - web
+    - refresh_token
+    - offline_access
+
+## Supported identities {#supported-identities}
+
+Salesforce CRM supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+| state | State ||
+| country | Country ||
+| number | Mobile Phone ||
+| firstName | First Name ||
+| postalCode | Mailing Postal Code ||
+| city | Mailing City ||
+| lastName | Last Name ||
+| email |Salesforce Email ID|Note that *Salesforce CRM* does not support hashed email addresses, hence only plain text data without transformation is sent to the destination.|
```

Comment:
Apologies for the back-and-forth on this one, but we need to make a change. 

`email` is the only identity in this table, all the other items are **attributes**. 

Regarding the other attributes, are these the target fields available to users? (similar to this example from another destination)

![Screen Shot 2022-06-02 at 11 45 35 AM](https://user-images.githubusercontent.com/19704189/171669211-2ee95718-4bff-4f8e-8167-c89cc67257d0.png)

Reply:
Many thanks for the clarification. Was a tad concerned about this one. Yes these fields are available to users. There would be more possible but it depends on what requires to be pushed.